joint_trajectory_controller/test/joint_trajectory_controller_wrapping_test.cpp

```diff
@@ -29,8 +29,8 @@
 
 #include <algorithm>
 #include <cmath>
+#include <memory>
 
-#include <boost/shared_ptr.hpp>
 #include <boost/thread/mutex.hpp>
```

Comment:
Any thoughts on replacing the boost mutex's during the c++11 migration as well?

Reply:
I'm all up for it!

Reply:
I've done it ;)